### DISCUSSION TITLE
CT Experimental Setup Plotter

### DISCUSSION THREAD

**Discussion by oobie2142** [astra-toolbox/astra-toolbox#320] [General]
Hello, 

Is there a plot function that draws the geometric set up of a CT reconstruction? I remember seeing something on a blog post showing examples of ASTRA reconstructions and they used something like `astra_plot`. 

Looking through the repository I saw something inside the Matlab folder, but not in the Python. 

Any help would be appreciated

Thanks!

**Comment by askorikov** [comment]
One suggestion would be to look into [Tomosipo](https://github.com/ahendriksen/tomosipo) library that can do geometry plotting and animation.
